src/components/Image/Image.tsx

```diff
@@ -7,34 +14,104 @@ export interface SourceSet {
 
 export type CrossOrigin = 'anonymous' | 'use-credentials' | '' | undefined;
 
+export interface State {
+  hasError: boolean;
+  hasLoaded: boolean;
+}
+
 export interface Props extends React.HTMLProps<HTMLImageElement> {
   alt: string;
+  className: string;
```

Comment:
className was being passed down, now we need it in here as well for the visibility.